test/players.js

```diff
@@ -0,0 +1,118 @@
+const assert = require('assert');
+const supertest = require('supertest');
+
+const app = require('../express/routes');
+const models = require('../sequelize/models');
+
+describe('GET /players', () => {
+    it('should return all players', async () => {
+        await supertest(app)
+            .get('/players')
+            .expect(200);
+    });
+});
+
+describe('GET /players/:id', () => {
+    it('should return player', async () => {
+        const testPlayer = await models.player.findOne();
```

Comment:
how about clearing DB before each test case?

Reply:
I'll create a sync test db later

Reply:
any reason why you are not able to do now?